app/controllers/vluchte_controller.rb

```diff
@@ -0,0 +1,42 @@
+# Entry point for the webhook resource
+class VluchteController < ApplicationController
+  before_action :validate, only: [:post]
+
+  def post
+    data = JSON.parse(request.body.read)
+    not_found = []
+    data.each do |bill|
+      user = User.find_by(email: bill['user']['email'])
+      unless user&.active?
+        not_found << bill['user']['email']
+        next
+      end
+
+      UserMailer.mail_vluchte_bill(bill, user).deliver
```

Comment:
Hier moeten we eigenlijk iets asyncs op vinden, als dit lang duurt gaat de request timeout geven. 

Misschien hier eens naar kijken: https://www.mikeperham.com/2022/10/27/sidekiq-7.0-embedding/ 

Anders een record aanmaken en eens in de zoveel tijd via een cron versturen.